modules/etcd-cert-alerts-metrics-signer.adoc

```diff
@@ -11,6 +11,8 @@ Two alert types inform users about pending `etcd` certificate expiration:
 `etcdSignerCAExpirationWarning`:: Occurs 730 days until the signer expires.
 `etcdSignerCAExpirationCritical`:: Occurs 365 days until the signer expires.
 
+The signer certificate authority expiration dates are tracked in the namespace.
```

Comment:
what does that mean? The expiration dates are stored on the annotation of the respective secret that expires